.github/workflows/update-3rdparty.yml

```diff
@@ -0,0 +1,254 @@
+name: Update 3rdparty sources
+on:
+  push:
+#     branches:
```

Comment:
This is to be uncommented before merging so it only runs on release branches. It's commented out for now so we can test.